src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -681,5 +690,15 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         email.setReplyTo(Config.EMAIL_REPLYTO);
         return email;
     }
-
+    
+    private String getCoOwnersEmailsList(List<InstructorAttributes> instructors){
+        String coOwnersEmailsList = "";
+        for (InstructorAttributes instructor: instructors) {
+            if(instructor.hasCoownerPrivileges()) {
+                coOwnersEmailsList += instructor.getEmail()+", ";
+            }
```

Comment:
@damithc can you explain further what you mean by this? what do you mean by `nesting of the main path`?